active/NavigationView/NavigationViewTemplateSettings/PaneTitle.md

```diff
@@ -0,0 +1,45 @@
+# Background
+PaneTitle is added to NavigationView in RS4, but it's a String. Since RS5, we introduced PaneHeader(UIElement) to deprecate PaneTitle. If both PaneTitle and PaneHeader are provided by customer, PaneHeader is used.
+
+This API update addes PaneTitle [Template-Settings](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/template-settings-classes) to the NavigationView control.
+
+# Description
+Provides calculated PaneTitle that can be referenced as TemplatedParent sources when defining templates for a NavigationView control. Not intended for general use.
+
+# Examples
+
+```
+[webhosthidden]
+unsealed runtimeclass NavigationViewTemplateSettings : Windows.UI.Xaml.DependencyObject
+{
+    ...
+
+    {
+        String PaneTitle{ get; };
+        static Windows.UI.Xaml.DependencyProperty PaneTitleProperty{ get; };
+    }
+}
+```
+
+# Remarks
+This is intended for use in the style of the `NavigationView` control. Sample usage:
+
+```xaml
+    <Style TargetType="local:NavigationView">

```

Comment:
Should be a template example.

Reply:
I don't understand? TemplateSettings.PaneTitle example is always in the style

Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.PaneTitle}"

Reply:
It's used in the ControlTemplate, and the ControlTemplate is typically part of the default Style. (That Text property is on a TextBlock in a ControlTemplate.) PersonPictureSettings (#13) is a good reference.